### ISSUE TITLE
Update permissions to remove ability to manually change expiration dates (NN)

### ISSUE THREAD

**Issue by atdservicebot** [cityofaustin/atd-data-tech#12273]
> What application are you using?

AMANDA

> Describe the problem.

Update permissions to remove ability to manually change expiration dates

> How soon do you need this?

Soon — Within a week 

> Is there anything else we should know?

Individuals currently have permissions they shouldn't

> Requested By
Kimberly P.

[Attachment](https://api.knack.com/v1/applications/595d00ebd315cc4cb98daff4/download/asset/645d4c3d9fa5d90028fb6954/copyofvalidprivilegeuser.xlsx) (13.86kb)

Request ID: DTS23-107850

**Comment by TracyLinder** [comment]
Currently everyone has permissions to manually update the folder expiration date. Paloma included a list in the description with users who shouldn't have this permission. 

Nate is working with Jenice to discuss similar issues with the UC folder. He'll make updates when he finishes talking to Jenice.

**Comment by Nadin-Nader** [comment]
This has been completed in all environments. I removed the Issue Expiry Security Group permission for everyone on the list provided except for the following people:

LOZANOJ Jaclyn Lozano
HOANGBRI Hoang Brian
PEPPERGR Greg Pepper

I also deactivated the following accounts that were no longer a part of COA, per the list:
MICHELLP Peter Michell
MATRANGAJ Jonathon Matranga
ROYL Laura Roy
LEWALLEN Isaiah Lewallen
SLAUSOND Doug Slauson
I will send a request to CTM to have the user accounts removed from AMANDA. I will also send Kim and Kyle an email with this message and close the card.
